apps/gallery/js/ImageEditor.js

```diff
@@ -119,7 +119,7 @@ var exposureSlider = (function() {
   thumb.addEventListener('pan', function(e) {
     // Handle delta so that slider moves correct way
     // when user drags it for RTL locales
-    var delta = navigator.mozL10n.language.direction === 'ltr' ?
+    var delta = document.documentElement.dir === 'ltr' ?
```

Comment:
Does document.documentElement.dir change correctly when the user switches locales?